frontend/packages/ux-editor/src/utils/formLayout.ts

```diff
@@ -18,7 +18,10 @@ import i18next from 'i18next';
 
 const { addFormComponent, addFormContainer, addWidget, updateActiveListOrder } = FormLayoutActions;
 
-export function convertFromLayoutToInternalFormat(formLayout: any[], hidden: any): IInternalLayouts {
+export function convertFromLayoutToInternalFormat(
+  formLayout: any[],
+  hidden: any
```

Comment:
We really need to remove the extensive use of `any` in the solution. This is not the PR for it, but it really make quality bleed.